plugins/sigma.renderers.customShapes/sigma.renderers.customShapes.canvas.js

```diff
@@ -32,10 +27,7 @@
         image = document.createElement('IMG');
```

Comment:
I've tested it and it works, but Chrome will prevent you to load images from file://. Use `npm start` to browse the examples, e.g. http://localhost:8000/examples/plugin-customShapes-hybrid.html